Tensile/KernelWriterAssembly.py

```diff
@@ -3992,7 +4200,17 @@ def localReadInc(self, kernel, tP):
           "lr%s += %u (LSU*(MT+PAD)*bpe)"%(tP["tensorChar"], inc) )
     else:
       if tP["localReadInstruction"].numOffsets == 1:
-        tP["localReadOffset"] += kernel["LocalSplitU"]*(kernel["MacroTile%u"%tP["tensorIdx"]] + kernel["LdsPad%s"%tc])
+        ldl = kernel["LocalDotLayout"]
+        if ldl > 1:
+          #jgolds
+          #HACK just hard coding to verify it works for the case I am testing
+          partialInc = 8    # in elements
```

Comment:
Looks like we are missing my fixes for this. In my tree I see:
#partialInc = 8
tt= tP["tt"]
partialInc = kernel[tt]